client/src/helpers/stardust/preExpandedConfig.ts

```diff
@@ -90,6 +104,30 @@ export function getInputsPreExpandedConfig(inputs: IInput[], unlocks: Unlock[],
     return inputsPreExpandedConfig;
 }
 
+function getReferencedAddresses(
+    inputs: IInput[],
+    unlocks: Unlock[],
+    idx: number,
+    bech32Hrp: string,
+): { referencedStateControllerAddress: string; referencedAliasAddress: string } {
+    let referencedStateControllerAddress = "";
+    let referencedAliasAddress = "";
+    const unlock = unlocks[idx];
+    if (unlock.type === UnlockType.Alias) {
+        // get the referenced alias output
```

Comment:
Each comment can be merged into it's line by just naming the variable differently :D 
```
// get the referenced alias output
const referencedAliasInput = inputs[(unlock as AliasUnlock).reference];
```
to
```
const referencedAliasOutput = inputs[(unlock as AliasUnlock).reference];
```